src/openforms/submissions/tests/test_tasks_registration.py

```diff
@@ -31,39 +24,12 @@ def test_source_reference_from_registration_result(self, mock_generate):
             registration_result={"zaak": {"identificatie": "AEY64"}},
         )
 
-        obtain_submission_reference(submission.id)
+        set_submission_reference(submission)
```

Comment:
can't you use the method here instead?

Reply:
I removed the method (because of the other comment), so I'm leaving these!